### PR TITLE
Bring back lint:ccip in precommit

### PR DESCRIPTION
**Pull request by matYang**
## Motivation
Resurrecting `lint:ccip` in `make ccip-precommit`, which was removed in the previous merge from main. I'm not sure if that was done by mistake. 
Happy to drop this PR if it was intentional.

### CODE REVIEW

**Review by mateusz-sekara — APPROVED** [review]

**Review by RensR — CHANGES_REQUESTED** [review]
lint:ccip is just running solhint. It should be removed

**Review by RensR — APPROVED** [review]

File: contracts/package.json [hunk 1]
```diff
@@ -18,8 +18,7 @@
     "prepublishOnly": "pnpm compile && ./scripts/prepublish_generate_abi_folder",
     "publish-beta": "pnpm publish --tag beta",
     "publish-prod": "npm dist-tag add @chainlink/contracts-ccip@0.7.6 latest",
-    "lint:ccip": "solhint --config ./src/v0.8/ccip/.solhint.json --ignore-path .ccip-solhint-ignore --max-warnings 0 \"./src/v0.8/ccip/**/*.sol\"",
-    "solhint": "solhint --max-warnings 404 \"./src/v0.8/**/*.sol\""
+    "solhint": "solhint --ignore-path .solhintignore --max-warnings 404 \"./src/v0.8/ccip/**/*.sol\""
```
Comment (RensR) [inline contracts/package.json:21]: This is used by CI, let's not change this

**Review by RensR — CHANGES_REQUESTED** [review]
Currently it would change CI and it would allow 404 warnings in CCIP files

**Comment by matYang** [comment]
changing this to a backlog item on reduce warnings from CCIP tests

**Pull request closed without merging**